nodepool/setup_nodepool.yml

```diff
@@ -29,7 +29,7 @@
   serial: 1
   vars:
     nodepool_file_nodepool_yaml_src: "{{ lookup('env', 'WORKSPACE') }}/rpc-gating/nodepool/files/nodepool.yml"
-    nodepool_git_version: 3de0b20faa6f90437eb62c56ec23c59c6e15b5fa # HEAD of 'master' as of 20 Mar 2018
+    nodepool_git_version: Iff97c70589ebc41f33d114f6fd978aea8219b6b1 # HEAD of 'master' as of 24 July 2018
```

Comment:
Maybe I'm doing something different to you, but I can't find this SHA in https://github.com/openstack-infra/nodepool - the current head is https://github.com/openstack-infra/nodepool/commit/67824d8e64250ede0cb4a3ebe344412f5cba2218 for me.

Reply:
WOW, I dunno how, but I got the ID from what seems to be not the git commit but from the change id in the git comment.  Weird. Will fix.